go-controller/pkg/ovn/gateway_init.go

```diff
@@ -127,6 +127,7 @@ func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*n
 		}
 	}
 
+	// RIP OUT
```

Comment:
I'm not sure what this is...If it is a TODO you need to have that label, preferably in a format like "// TODO(astoycos): something todo"